docs/Makefile

```diff
@@ -0,0 +1,28 @@
+PANDOC = pandoc
+
```

Comment:
How does this cope with math?

Reply:
Generating PDFs via HTML means that they will have the same math limitations as HTML conversion, i.e. we are limited to characters supported by Unicode. This means that we can support basic math, but more complex math display (like fractions, summations etc.) is not possible. 

So, for example, the following would work:
$$a_{min} = (b_1 \times 5)^2 $$

But the following would not work
$$\frac{5}{8} \times 8 = 5$$

Reply:
I tried it (had to install `wkhtmltopdf`) and the first differences I notice are:
1. The PDF generated via LaTeX has a PDF table of contents while the one generated from HTML doesn't; this might make a difference for longer documents.
2. The PDF generated via HTML has visible links while the one generated via LaTeX has links, but there's no way to know until you hover your mouse over the link. (That can be easily fixed by passing `-V colorlinks=true` with my version of pandoc (2.5 from Ubuntu 20.04).)

I'm not sure math support is going to make much of a difference: we are generating both PDF and HTML, so we can't use math that HTML doesn't support anyway. Or am I missing something?